py-write-code.md

```diff
@@ -71,17 +71,17 @@ In this example, if the user selects the result, the `open_url` method will be c
 ### 5. Create an additional context menu
 **def context_menu(self, data):**
 
-This method creates a context menu for your results, where the user can carry out additional tasks when they go to the context menue via pressing `Shift + Enter`. A context menu could be helpful if you want some tasks specific to your returned results, for example the Explorer plugin would return a list of file results, and when going to the context menu of one of the result users can select to copy the file.
+This method creates a context menu for your results, where the user can carry out additional tasks when they go to the context menu via pressing `Shift + Enter`. A context menu could be helpful if you want some tasks specific to your returned results. For example, the Explorer plugin would return a list of file results, and when going to the context menu of one of the result users can select to copy the file.
 
-To attach a method to your context menu result, do the same as for normal results where you define a JsonRPCAction item with the method and parameters you want to call and pass through. In this case the context menu will simply open the HelloWorldPython plugin's GitHub repo.
+To attach a method to your context menu result, do the same as for normal results where you define a JsonRPCAction item with the method and parameters you want to call and pass through. In this case, the context menu will simply open the HelloWorldPython plugin's GitHub repo.
 
 ### 6 Result score
-The `score` field provides the ability to assign a weight to your score, the higher the score is, the higher the result from the plugin would show in flow's result list. The range in which you assign the score is usually between 0-100. You can keep it as 0 if your plugin generally uses an action keyword to trigger, but if you are using a global action keyword - `*` then the average weight for a plugin would be 50. Additionally users can also tweak the score via Flow's plugin setting as well.
+The `score` field provides the ability to assign a weight to your score; the higher the score is, the higher the result from the plugin would show in flow's result list. The range in which you assign the score is usually between 0–100. You can keep it as 0 if your plugin generally uses an action keyword to trigger, but if you are using a global action keyword - `*` then the average weight for a plugin would be 50. Additionally, users can tweak the score via Flow's plugin setting as well.
```

Comment:
what's the difference between `-` with `–`. I don't think the later one is easy to type?

Reply:
Also the use of `-` as appositive (for global action keyword) seems to make it a bit weird...maybe make it with a parenthesis?

Reply:
The former is a hyphen, the latter is an en-dash. It's slightly wider than hyphens, and is usually used in number ranges in typography in English. Do you want me to return the hyphens?

As for the action keyword, yes, wrapping it in parentheses instead does make sense.

Reply:
if that's the correct use case of en-dash then it is fine. Thanks for the information.